WrightTools/data/_channel.py

```diff
@@ -8,11 +8,13 @@
 
 import h5py
 
+from typing import TypeVar
+
 from .. import kit as wt_kit
 from .._dataset import Dataset
 
 __all__ = ["Channel"]
-
+num_type = TypeVar("num_type", int, float, complex)
```

Comment:
Perhaps for future, but we could consider making channel, variable, etc, Generic types. Personally, wish to wait to see how numpy/h5py handle such things

Reply:
Also:

https://www.python.org/dev/peps/pep-0484/#the-numeric-tower